src/main/java/com/test/sb/service/mapper/UserMapper.java

```diff
@@ -0,0 +1,33 @@
+package com.test.sb.service.mapper;
+
+import com.test.sb.dto.ReviewDto;
+import com.test.sb.model.Role;
+import com.test.sb.model.User;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper implements GeneralMapper<User> {
+    private static final Set<Role> DEFAULT_ROLE = Set.of(Role.of("USER"));
+
+    @Value("${default.user.password}")
+    private String defaultPassword;
+
+    private final PasswordEncoder encoder;
```

Comment:
we should create a separate class in our project where we add all the entities that were parsed from the file to the database

in this class we should fetch the role by rolename from the database (this role should have id already), and then set this role to each user

so, let's create that class and remove setting the roles and password from here